plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -177,24 +177,22 @@ export default function TableChart<D extends DataRecord = DataRecord>(
       const groupByValues = Object.values(filters);
       setDataMask({
         extraFormData: {
-          append_form_data: {
-            filters:
-              groupBy.length === 0
-                ? []
-                : groupBy.map(col => {
-                    const val = filters?.[col];
-                    if (val === null || val === undefined)
-                      return {
-                        col,
-                        op: 'IS NULL',
-                      };
+          filters:
+            groupBy.length === 0
```

Comment:
looks like EchartsPie data, may be we can create common utils if it relevant also for other charts?

Reply:
Good idea - however, let's leave that for a follow-up PR (we should avoid doing bycatch fixes/refactors)

Reply:
👍